digital_land/cli.py

```diff
@@ -287,16 +286,57 @@ def collection_add_source_cmd(ctx, collection, endpoint_url, collection_dir):
     "expectations-dataset-checkpoint",
     short_help="runs data quality expectations against a dataset sqlite file",
 )
-@click.option("--data-path", help="path to the sqlite3 dataset", required=True)
-@click.option("--output-dir", help="directory to store results", required=True)
-@click.option("--specification-dir", help="checkpoint to run", required=True)
-def expectations_run_dataset_checkpoint(data_path, output_dir, specification_dir):
+@click.option(
+    "--dataset",
+    type=click.STRING,
+    help="the dataset which is stored in the file path, the checkpoint is ran for one dataset at a time",
+    required=True,
+)
+@click.option(
+    "--file-path",
+    type=click.Path(),
+    help="path to the sqlite3 dataset that contains the dataset data",
+    required=True,
+)
+@click.option(
+    "--log-dir",
+    type=click.Path(),
+    help="directory to store expectation logs. an expectation directoy will be ceated here",
+    required=True,
+)
+@click.option(
+    "--configuration-path",
+    type=click.Path(),
+    help="path to the configuration sqlite file",
+    required=True,
+)
+@click.option(
+    "--organisation-path",
+    type=click.Path(),
+    help="path to the organisation data for the organisation class",
+    required=True,
+)
+@click.option(
+    "--specification-dir",
+    type=click.Path(),
+    help="directory  containing the specification",
+    required=True,
+)
+def expectations_run_dataset_checkpoint(
+    dataset,
+    file_path,
+    log_dir,
+    configuration_path,
+    organisation_path,
+    specification_dir,
+):
     from digital_land.expectations.commands import run_dataset_checkpoint
```

Comment:
Does this need to be imported here, rather than the top of the file?

Reply:
I think it was here to stop spatialite being imported but that doesn't matter anymore